sdk/purview/purview-datamap-rest/src/clientDefinitions.ts

```diff
@@ -1334,8 +1332,8 @@ export interface Routes {
   (path: "/search/autocomplete"): DiscoveryAutoComplete;
   /** Resource for '/atlas/v2/lineage/\{guid\}' has methods for the following verbs: get */
   (path: "/atlas/v2/lineage/{guid}", guid: string): LineageGet;
-  /** Resource for '/lineage/\{guid\}/next' has methods for the following verbs: get */
-  (path: "/lineage/{guid}/next", guid: string): LineageGetNextPage;
+  /** Resource for '/lineage/\{guid\}/next/' has methods for the following verbs: get */
+  (path: "/lineage/{guid}/next/", guid: string): LineageGetNextPage;
```

Comment:
not sure if it would have impact on the end user?

Reply:
this should be correct 
![Uploading image.png…]()

Reply:
TypeSpec behavior change.